wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/examples.json

```diff
@@ -22,6 +22,16 @@
     "gradlebase": "java",
     "mainclass": "Main"
   },
+  {
+    "name": "Arcade Drive",
+    "description": "An example program which demonstrates the simplest autonomous and teleoperated routines.",
```

Comment:
I don't think this description is the best. Arcade Drive or (Shooter Style) is the system of driving where one axis is throttle, with the other being turn. "Simplest" could mean many of things.

Reply:
I'll clarify the description. Thanks.

Reply:
Does this description fit better?